### ISSUE TITLE
color-blind mode

### ISSUE THREAD

**Issue by CelticMinstrel** [wesnoth/wesnoth#6148]

**Comment by cooljeanius** [comment]
This was something @Zireael07 (a.k.a. "Chess") suggested:
<img width="745" alt="Screen Shot 2021-10-03 at 12 37 17" src="https://user-images.githubusercontent.com/1388660/135763220-a78892f6-2493-4893-8bf4-7e220ec85eb3.png">

**Comment by Zireael07** [comment]
nope, I'm not chess on Discord :) It just proves great minds think alike as s/he mentioned it in Discord at roughly the same time I did on GH ;)

**Comment by cooljeanius** [comment]
> nope, I'm not chess on Discord :) It just proves great minds think alike as s/he mentioned it in Discord at roughly the same time I did on GH ;)

oh sorry nvm then

**Comment by cooljeanius** [comment]
Color-Blindness Accessibility Guide from "Can I Play That?": https://caniplaythat.com/2020/01/29/color-blindness-accessibility-guide/
